ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/ArrayDecoderTest.kt

```diff
@@ -95,9 +176,44 @@ class SimpleArrayDecoderTest {
                 ]
                 """
         )
-        assertEquals(listOf<Long?>(null, 1), testWithMultilineArray.field)
+        assertEquals(listOf<Long?>(1, 2, 3), testWithMultilineArray.field)
+
+        testWithMultilineArray = Toml.decodeFromString(
+            """
+                field = [
+                    1,
+                    null,
+                    3
+                ]
+                """
+        )
+        assertEquals(listOf<Long?>(1, null, 3), testWithMultilineArray.field)
     }
 
+    @Test
+    fun testMultilineArraysWithComments() {
```

Comment:
Can you please add the test, where there is a missing ']' symbol in the end of multiline string. 
I think we should raise an error in this case